openedx_tagging/core/tagging/models/base.py

```diff
@@ -241,7 +241,7 @@ class Taxonomy(models.Model):
             "Indicates whether this taxonomy should be visible to object authors."
         ),
     )
-    # External ID that should only be used on import/export.
+    # Export ID that should only be used on import/export.
```

Comment:
nit: I don't think this comment should change.. the point of this field is that it's an externally valid identifier for the taxonomy, so keeping the word "External" is fine.

Reply:
Reverted: ac7bf0e66ce4532eb88f424258d2fd338fa06925
I thought it was a typo, but after reading it again, I agree with you.